bigbluebutton-html5/imports/ui/components/user-info/component.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { defineMessages, intlShape } from 'react-intl';
+
+import Button from '/imports/ui/components/button/component';
+import Modal from '/imports/ui/components/modal/simple/component';
+import { makeCall } from '/imports/ui/services/api';
+
+import Service from './service';
+
+import { styles } from './styles';
+
+const propTypes = {
+  intl: intlShape.isRequired,
+};
+
+const intlMessages = defineMessages({
+  title: {
+    id: 'app.user-info.title',
+    description: 'User info title label',
+  },
+});
+
+class UserInfo extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  renderUserInfo(UserInfo) {
+    const userInfoList = UserInfo.map((user, index, array) => {
```

Comment:
Are we able to split this into rows renderers so that change on one userInfo document does not trigger recalculation of the entire table?